setup.py

```diff
@@ -99,7 +99,6 @@ def checkProgramIsInstalled(self, program, args, where_to_download,
     author='Fidel Ramirez, Vivek Bhardwaj, Björn Grüning',
     author_email='deeptools@googlegroups.com',
     packages=['hicexplorer'],
-    package_data={'': ['config/deeptools.cfg']},
     scripts=['bin/findRestSite', 'bin/hicBuildMatrix', 'bin/hicCorrectMatrix',
```

Comment:
:) I hope it is now fixed. The version number looks wired and is not supported with newer pip versions. We need to change this before pushing to pypi.